spec/13-functions.md

```diff
@@ -74,6 +74,7 @@ A function is called via the function-call operator [`()`](10-expressions.md#fun
 
   <i>return-type:</i>
     : <i>type-declaration</i>
+    : void
```

Comment:
Oh wait, it actually does mention that in the lexical structure section. I'll update my patch to update that for void.